core/metacling/src/TCling.cxx

```diff
@@ -1348,7 +1348,6 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
                   ModulesPreloaded.push_back(ModuleName);
             }
          }
-         LoadModules(ModulesPreloaded, *fInterpreter);
```

Comment:
Could the commit log explain why this is no longer needed?

Reply:
You're right. This is totally wrong